Dockerfiles/manifests/cluster-checks-runners/agent-secret.yaml

```diff
@@ -0,0 +1,9 @@
+# This file has been generated by `helm template datadog-agent stable/datadog` from datadog/templates/agent-secret.yaml. Please re-run `generate.sh` rather than modifying this file manually.
+apiVersion: v1
+kind: Secret
+metadata:
+  name: datadog-agent-cluster-agent
+  labels: {}
+type: Opaque
+data:
+  token: "ZWNFVU9vVm92R0xpOXg5Uk5mNE1BZ05WOFZQWUV4UmE="
```

Comment:
random token?

Reply:
Yes, this one is random because it’s what helm does when no other value is specified.
The drawback is that each time we launch `generate.sh`, even without any change in the chart, a new random token will be generated and will make `git` flag the file as “modified”.